codec/src/main/java/io/netty/handler/codec/DefaultTextHeaders.java

```diff
@@ -579,7 +428,8 @@ private TextHeaders addEscapedValue(CharSequence name, CharSequence escapedValue
         }
 
         private <T> CharSequence commaSeparate(CsvValueEscaper<T> escaper, T... values) {
-            StringBuilder sb = new StringBuilder(values.length * DEFAULT_VALUE_SIZE);
```

Comment:
> who is going to configure it?

Anyone who is using features that depend upon it and may have a more ideal number for their use case. More likely to be noticed if it is at the head of the file bcz it is just a guess...

I'm fine with leaving it...bigger fish to fry...